src/dialect/hive.rs

```diff
@@ -36,4 +36,8 @@ impl Dialect for HiveDialect {
             || ch == '{'
             || ch == '}'
     }
+
+    fn supports_filter_during_aggregation(&self) -> bool {
```

Comment:
I believe we would want to add this to `src/dialect/postgres.rs` as well since postgresql supports `FILTER` also https://www.postgresql.org/docs/current/sql-expressions.html

Reply:
Done